About/About/faq.md

```diff
@@ -1,14 +1,13 @@
-
 # FAQ
 
-## How do open source community health metrics help me understand whether an open source project is healthy?
+# How do open source community health metrics help me understand whether an open source project is healthy?
```

Comment:
The questions should continue to be a sub-heading with `##`.

Reply:
Done...I made the changes as suggested in the review!!

Reply:
Thank you!